django/db/migrations/serializer.py

```diff
@@ -270,62 +271,64 @@ def serialize(self):
         return "uuid.%s" % repr(self.value), {"import uuid"}
 
 
+def is_deconstructs(value):
+    unsuitable_types = (models.Field, models.manager.BaseManager, Operation, type)
+    return not isinstance(value, unsuitable_types) and hasattr(value, 'deconstruct')
+
+
+class RegisterSerialize:
```

Comment:
How about `class Serializer` and method `register()` so that registering looks like `Serializer.register()`?